### ISSUE TITLE
initialHTML hijacks the autofocus

### ISSUE THREAD

**Issue by dvnrsn** [CanopyTax/bandicoot#58]
If I provide the `initialHTML` prop, bandicoot will autofocus on the contentEditable, stealing from other `<inputs` with `autoFocus`

[codesandbox](https://codesandbox.io/s/musing-chandrasekhar-2iodw?fontsize=14)

**Comment by dvnrsn** [comment]
I tried addressing both `.focus()` invocations. It seems that the useEffect:
```js
if (!hasSetInitialHTML && props.initialHTML) {
```
invokes `setHTML` and from there `focus`. However, even when the `.focus()` code of `focus` function is removed, we still have a focus on `<div contentEdiatble`

**Comment by dvnrsn** [comment]
It seems that `emptyEditor()` is the culprit...
in particular this line which is probably non-negotiable:
```js
selection.addRange(range)
```

**Comment by internettrans** [comment]
I forget why we call emptyEditor() inside of setHTML(), but I think it’s for undo/redo.

Additionally we call focus() inside of setHTML, which would also mess with Auto focus.

Maybe it would make sense to change this line:

https://github.com/CanopyTax/bandicoot/blob/b541feda5917932b4f66b5f19c8173446c91d1d9/src/rich-text-editor.component.js#L124

And pass in a Boolean flag to setHTML that tells it not to empty the editor and forcibly focus? Such that the editor is only emptied and focused for subsequent setHTML calls, but not for the initial html.

If we go down that approach, we probably would want to add an autoFocus prop to RichTextEditor so that the content editable element can be auto focused, if desired.

This change would be a breaking change because the editor would no longer autoFocus by default, but now require a prop to autoFocus. But that would be acceptable I think - we’d just bump the major version.

Thoughts?

**Comment by dvnrsn** [comment]
fixed by #65

**Comment by dvnrsn** [comment]
Argh, why is this not working? It seems to be working locally for me with `yarn link` but then fails when I use the npm package. Could my publish have failed? [codepen](https://codesandbox.io/s/musing-chandrasekhar-2iodw?fontsize=14)
